config.py

```diff
@@ -89,31 +89,47 @@
 
 DIVERT_TO_PRIVATE = ('help', )
 
-ROOMS_TO_JOIN = (
-    'coala',
-    'coala/offtopic',
-    'cobot-test',
-    'corobo',
-    'devops',
-    'community',
-    'coala/gsoc',
-    'coala/maintainers',
-    'coala-bears',
-    'bearship',
-    'gci',
-    'cobot',
-    'performance',
-    'documentation',
-    'conferences',
-    'coala/workshops',
-    'coala/artwork-corner',
-    'freelancers',
-    'editor-plugins',
-    'depman',
-    'ast',
-    'aspects',
-    'community'
-)
+if BACKEND == 'Gitter':
+    ROOMS_TO_JOIN = (
+        'coala',
+        'coala/offtopic',
+        'cobot-test',
+        'corobo',
+        'devops',
+        'community',
+        'coala/gsoc',
+        'coala/maintainers',
+        'coala-bears',
+        'bearship',
+        'gci',
+        'cobot',
+        'performance',
+        'documentation',
+        'conferences',
+        'coala/workshops',
+        'coala/artwork-corner',
+        'freelancers',
+        'editor-plugins',
+        'depman',
+        'ast',
+        'aspects',
+        'community'
+    )
+
+if BACKEND == 'Zulip':
+    ROOMS_TO_JOIN = (
+        'announce',
+        'coala',
+        'coala/maintainers',
+        'coala-bears',
+        'corobo',
+        'depman',
+        'gci-mentors-2018'
```

Comment:
There are more rooms in Zulip, even rooms which are the same as Gitter, e.g. `gci` and `ast`-- look for them all.

Reply:
I can't see any other rooms.

Reply:
Did you look at all streams, even the ones you are not subscribed too ?

Reply:
I found all the streams! I had to click search. It should soon be done.